src/qidenticon.py

```diff
@@ -9,7 +9,7 @@
 Return a PIL Image class instance which have generated identicon image.
 ``size`` specifies `patch size`. Generated image size is 3 * ``size``.
 """
-
+import six
```

Comment:
Excuse me, I should write `from six.moves import range`. Because in that case you just remove this line when dropping python2 support. It's better to look into the code for similar cases than rely on my hints.